src/app/question-authoring-page/question-authoring-page.component.html

```diff
@@ -0,0 +1,64 @@
+<clr-main-container class="main-container">
+<clr-header class="header header-1">
+    <div class="branding">
+        <a routerLink="/home"  routerLinkActive="active" class="nav-link">
+            <img src="assets/dojo-transparent-white-outline.svg" alt="A dojo" class="mtdj__logo">
+            <span class="title">Math Dojo</span>
+        </a>
+    </div>
+
+</clr-header>
+<div class=" question-form-wrapper" >
```

Comment:
It’ll help with our styling if we make use of some of the Clarity Design System elements and approach here: https://clarity.design/documentation/forms